doc/man5/config.pod

```diff
@@ -271,7 +271,7 @@ provider will be activated. Conversely, setting this value to no, off, false, or
 or uppercase. Setting activate to any other setting, or omitting a setting
 value will result in an error.
 
-= item B<soft_load>
+=item B<soft_load>
```

Comment:
This should be a separate bugfix PR (or at least a separate commit) so it can be cherry picked to stable branches.